### ISSUE TITLE
Isolated Sphinx doc does not build due to missing pytools script

### ISSUE THREAD

**Issue by ben-howt** [BCG-Gamma/facet#343]
**Describe the bug**
Sphinx doc does not build due to missing pytools script `make_base` if `pytools` is not in same parent directory

**To Reproduce**
1. Clone the repo into a separate directory from any pytools clones
2. Build and activate the conda environment in `environment.yml`
3. From the `sphinx` directory, run `python make.py html`

```
Traceback (most recent call last):
  File "make.py", line 30, in <module>
    make()
  File "make.py", line 24, in make
    from make_base import make
ModuleNotFoundError: No module named 'make_base'
```

**Expected behavior**
Docs should build.

**Desktop (please complete the following information):**
 - OS: MacOS Big Sur
 - Version 1.2.x and 2.0.x

**Additional context**
Likely this is just a note in `contributing.md` to make sure that the remotes are all present before trying to build the docs.

**Comment by j-ittner** [comment]
@ben-howt this is by design, so we can maintain a single make script across all 3 associated repos. Open for alternative suggestions - any ideas how to handle better? Would it be sufficient to explain the approach better in the developer guidelines?

**Comment by ben-howt** [comment]
Thanks @j-ittner for the response; this makes sense to me as a design decision. I would suggest (and can raise a PR for) the following 2 changes:

* Add a note in `sphinx/source/contribution_guide.rst`
* Add a descriptive error if pytools is not found in the `sys.path.insert` segment of `sphinx/make.py`

**Comment by j-ittner** [comment]
Addressed by #345
